src/shared/components/home/instances.tsx

```diff
@@ -185,6 +187,10 @@ export class Instances extends Component<InstancesRouteProps, InstancesState> {
                 </td>
                 <td>{i.software}</td>
                 <td>{i.version}</td>
+                <td>
+                  {formatRelativeDate(i.updated_at ?? i.published_at)}
+                  {isWeekOld(new Date(i.updated_at ?? i.published_at)) && " 💀"}
```

Comment:
What does the skull mean?

Reply:
Dead instance / not contacted in a week. Maybe it'd be good to add a tooltip for that case.

Reply:
Not sure if a tooltip is really necessary as dead instances are already listed at the bottom with https://github.com/LemmyNet/lemmy/pull/5946. So I can also get rid of the emoji.